gateway/api/openapi/types.go

```diff
@@ -1115,6 +1118,9 @@ type GuardRailRuleResponse struct {
 	*/
 	Output map[string]any `json:"output"`
 
+	// List of connection IDs or names that this guardrail applies to
+	ConnectionIDs []string `json:"connection_ids" example:"pgdemo,redis-prod"`
```

Comment:
The example is incorrect, it should has uuid instead of names. Make sure to generate the openapi documentation as well